docs/src/modules/sandbox/StackBlitz.ts

```diff
@@ -0,0 +1,40 @@
+import SandboxDependencies from './Dependencies';
+import * as CRA from './CreateReactApp';
+import getFileExtension from './FileExtension';
+
+const createReactApp = (demo: {
+  title: string;
+  language: string;
+  raw: string;
+  codeVariant: 'TS' | 'JS';
+  githubLocation: string;
+  product?: 'joy-ui' | 'base';
+}) => {
+  const ext = getFileExtension(demo.codeVariant);
+  const { title, githubLocation: description } = demo;
+
+  const files: Record<string, string | object> = {
+    'index.html': CRA.getHtml(demo),
+    [`index.${ext}`]: CRA.getRootIndex(demo.product),
+    [`demo.${ext}`]: demo.raw,
+    ...(demo.codeVariant === 'TS' && {
+      'tsconfig.json': CRA.getTsconfig(),
+    }),
+  };
+
+  const { dependencies } = SandboxDependencies(demo);
+
+  dependencies['@babel/runtime'] = 'latest';
+  dependencies['@babel/core'] = 'latest';
```

Comment:
This is very strange, is it a regression from StackBlitz? Why would they need these two dependencies?

https://stackblitz.com/edit/react-ts-eedjlb?file=index.tsx 

```suggestion
```

Reply:
Seems like this is because StackBlitz sees our packages's dependencies. I'm not sure how to fix this unless making StackBlitz happy.

Reply:
You should fork it from our docs, this is an example https://stackblitz.com/run?file=demo.tsx

Reply:
it works without `@babel/core`: https://stackblitz.com/edit/react-z6ckyf?file=package.json

Reply:
Niice, it looks like you found out why, a regression then #32738

Reply:
`@babel/core` is removed.